src/Uno.UI/UI/Xaml/Internal/RootVisual.cs

```diff
@@ -34,32 +34,11 @@ internal partial class RootVisual : Panel
 	{
 		private readonly CoreServices _coreServices;
 
-		[ThreadStatic]
-		private static bool _canUnFocusOnNextLeftPointerRelease = true;
-
 		public RootVisual(CoreServices coreServices)
 		{
 			_coreServices = coreServices ?? throw new System.ArgumentNullException(nameof(coreServices));
 			//Uno specific - flag as VisualTreeRoot for interop with existing logic
 			IsVisualTreeRoot = true;
-#if __WASM__
-			//Uno WASM specific - set tabindex to 0 so the RootVisual is "native focusable"
-			SetAttribute("tabindex", "0");
-
-			AddHandler(
-				PointerCanceledEvent,
-				new PointerEventHandler((snd, args) => ProcessPointerCancelledWasm(args)),
-				handledEventsToo: true);
-#endif
-
-			AddHandler(
-				PointerPressedEvent,
-				new PointerEventHandler((snd, args) => ProcessPointerDown(args)),
-				handledEventsToo: true);
-			AddHandler(
-				PointerReleasedEvent,
-				new PointerEventHandler((snd, args) => ProcessPointerUp(args)),
-				handledEventsToo: true);
```

Comment:
FYI @MartinZikmund as this is likely going to cause merge conflicts with #12883

Reply:
Yes it did 😭